presto-manager-controller/src/main/java/com/teradata/prestomanager/controller/ApiScope.java

```diff
@@ -0,0 +1,14 @@
+package com.teradata.prestomanager.controller;
```

Comment:
Needs a license -- maybe you can add a build check for the license? Presto has one, I think.